src/Zikula/CoreBundle/Helper/PersistedBundleHelper.php

```diff
@@ -120,14 +120,15 @@ private function extensionIsActive(Connection $conn, string $class, string $type
      */
     public function addAutoloaders(ZikulaHttpKernelInterface $kernel, array $autoload = []): void
     {
+        $srcDir = $kernel->getProjectDir() . '/src/';
         if (isset($autoload['psr-0'])) {
             foreach ($autoload['psr-0'] as $prefix => $path) {
-                $kernel->getAutoloader()->add($prefix, $path);
+                $kernel->getAutoloader()->add($prefix, $srcDir . $path);
```

Comment:
@Guite something else to consider - with this change, the autoloader paths are all full paths. In the past they were relative. Maybe this doesn't matter. What does composer generate for the other vendors, full or relative? And if they are relative, couldn't you just put src/ in here?

Reply:
I looked into Composer's `autoload_psr4.php` and they are absolute there, too.